splash/request_middleware.py

```diff
@@ -72,6 +72,59 @@ def process(self, request, render_options, operation, data):
         return request
 
 
+class ContentTypeMiddleware(object):
+    """
+    Response middleware, aborts responses depending on the content type.
+    A response will be aborted (and the underlying connection closed) after
+    receiving the response headers if the content type of the response is not
+    in the whitelist or it's in the blacklist. Both lists support wildcards.
+    """
+    def __init__(self, verbosity=0):
+        self.verbosity = verbosity
+
+    @staticmethod
+    def contains(mime_set, mime):
+        for part1 in mime[0], '*':
+            for part2 in mime[1], '*':
+                if (part1, part2) in mime_set:
+                    return True
+        return False
```

Comment:
While globs are nice, I'm a bit worried that we have so many different matching languages in use: regexes for proxy profiles, AdBlockPlus filters for request filters, and now globs for Content-Type filters. What about using regexes?

If we go with globs, it is better to use https://docs.python.org/2/library/fnmatch.html.